decidim-meetings/app/controllers/decidim/meetings/admin/registrations_attendees_controller.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Meetings
+    module Admin
+      # This controller allows an admin to manage meeting registrations and
+      # attendances from a participatory space
+      class RegistrationsAttendeesController < Admin::ApplicationController
+        helper_method :registrations
+
+        def index
+          enforce_permission_to(:update, :meeting, meeting:)
```

Comment:
I see that you've created another permission for `:validate_registration_code`, maybe we should use it here as well?